eventloop_windows.go

```diff
@@ -203,6 +211,10 @@ func (el *eventloop) loopWake(c *stdConn) error {
 	//if co, ok := el.connections[c]; !ok || co != c {
 	//	return nil // ignore stale wakes.
 	//}
+	if c.conn == nil {
```

Comment:
ditto

Reply:
That condition needed since we can run Wake and Close in goroutines, so they will produce events in channel. Same for loopError.

Reply:
Should I replace `if c.conn == nil` to commented:
```go
if co, ok := el.connections[c]; !ok || co != c {
	return nil // ignore stale wakes.
}
```
here and in the `loopError`?

Reply:
It could be a better choice, so go ahead.

Reply:
But still, I can't get the point when users return `Close` in `React` and call conn.Wake at the same time.

Reply:
It’s not necessary to do that in react.
You can close conn externally and at a same time have enqueued actions in asyncQueue or like here - in goroutines waiting channel send.

Reply:
Well, I guess that could be the reason to add this check.